maintenance_equipment_tags/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2019 Creu Blanca
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Maintenance Equipment Tags",
+    "summary": """
+        Adds category tags to equipment""",
+    "version": "14.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Creu Blanca,Odoo Community Association (OCA)",
```

Comment:
```suggestion
    "author": "Creu Blanca,Odoo Community Association (OCA)",
    "maintainers": ["jaimearroyo7"],
```
I think you could add this

Reply:
Done!

Reply:
then mark as resolved this comment :)

Reply:
Put me please as maintainer, Jaime is no longer working with odoo